I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/finales2.rpy", line 39, in script
    jump cena3_sonho
ScriptError: could not find label 'cena3_sonho'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/finales2.rpy", line 39, in script
    jump cena3_sonho
  File "C:\Users\thoma\Downloads\renpy-8.3.6-sdk\renpy\ast.py", line 1712, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\thoma\Downloads\renpy-8.3.6-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'cena3_sonho'.

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
 
Tue Jul 22 22:14:55 2025
